Solutions/SIGNL4/Data Connectors/Logo/signl4.svg

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please reduce the size of logo. Logo should be less than 5kb. Please refer [guidelines](https://microsoft.sharepoint.com/:x:/r/teams/AzureSecurityInsights/_layouts/15/guestaccess.aspx?e=aETsaQ&share=EWs_HuWia89Ns0pc1jXRWQkB7aNXX6Zi1REQcKtH4gQp5Q&isSPOFile=1)

Reply:
Thank you for your review.
The SVG size is under 5kb now.
By the way, it seems I cannot open the link to the guidelines.